components/style/values/specified/outline.rs

```diff
@@ -10,8 +10,8 @@ use selectors::parser::SelectorParseErrorKind;
 use style_traits::ParseError;
 use values::specified::BorderStyle;
 
-#[derive(Clone, Copy, Debug, Eq, MallocSizeOf, Ord)]
-#[derive(PartialEq, PartialOrd, ToComputedValue, ToCss)]
+#[derive(Clone, Copy, Debug, Eq, MallocSizeOf, Ord, PartialEq, PartialOrd, ToComputedValue,
+         ToCss)]
```

Comment:
It also merges `#[derive(..)]` lines? Is there a way to prevent it from doing that?